### DISCUSSION TITLE
Laden gleicher Hörbuchtitel

### DISCUSSION THREAD

**Discussion by oerchen** [audiamus/BookLibConnect#138] [Q&A]
Hallo,

ich habe mal das erneute Laden des gleichen Titels abgewählt, wo kann ich dies wieder rückgängig machen? Ich möchte ein Hörbuch, jetzt mit anderem Sprecher herunterladen, es wird mir aber wegen des gleichen Titels nicht zum Download angeboten.

Beste Grüße

**Comment by audiamus** [comment]
Ein vollständiger "Resync" könnte helfen.

Die Einstellungen werden gespeichert unter `C:\Users\<User>\AppData\Local\audiamus\BookLibConnect\settings\usersettings.json`. Kann man dort ggf mit einem Texteditor bearbeiten. Die sollten sich jedoch auch alle über die Benutzeroberfläche ändern lassen.

**Comment by oerchen** [comment]
Vielen Dank, habe die Hörbücher gefunden.

In den Einstellungen inkl. usersettings.json kommt die Abfrage die ich hatte nicht vor. Aus der Erinnerung kam im ersten Popup der Hinweis auf gleiche Bezeichnung und die Abfrage ob überschreiben oder nicht. Danach kam ein 2. Popup mit der Frage ob diese Einstellung für alle weiteren so gespeichert werden soll oder nicht.

**Comment by oerchen** [comment]
Erledigt
